new/collapse-mixlib-install-in-tk.md

```diff
@@ -0,0 +1,171 @@
+---
+RFC: unassigned
+Title: Collapse Mixlib Install Code Paths in Test Kitchen
+Author: Patrick Wright <patrick@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Collapse Mixlib Install Code Paths in Test Kitchen
+Test Kitchen currently supports two code paths for configuring, downloading and installing Chef and ChefDK on test instances. Both code paths use Mixlib Install but are implemented differently. In the past year, significant effort has been put into Mixlib Install to make it the canonical source for interacting with Chef Software Inc's release infrastructure and generating bootstrap installation scripts (install.sh and install.ps1).
```

Comment:
(All of my comments are suggestions only; for the sake of conciseness, they aren't necessarily phrased that way.)

Establish language for talking about the old Mixlib Install API vs the new one.  For example, if you could say "significant effort has been put into Mixlib Install's new API to make the canonical source...", that's clearer and more compelling.

"canonical source" -> "canonical mechanism" or "canonical method"